cli/parse_pdfs.py

```diff
@@ -13,31 +12,32 @@
 
 import cloudpathlib.exceptions
 import fitz
-import layoutparser as lp
 import numpy as np
 import requests
 from cloudpathlib import CloudPath, S3Path
 from fitz.fitz import EmptyFileError
+from layoutparser import load_pdf, Layout, draw_box
+from layoutparser.models import Detectron2LayoutModel
+from layoutparser.ocr import TesseractAgent, GCVAgent
 from tqdm import tqdm
 
-sys.path.append("..")
-
 from src import config  # noqa: E402
 from src.base import (  # noqa: E402
     ParserInput,
     ParserOutput,
     PDFData,
     PDFPageMetadata,
 )
-from src.pdf_parser.pdf_utils.parsing_utils import (  # noqa: E402
-    LayoutDisambiguator,
+from src.pdf_parser.pdf_utils.disambiguate_layout import run_disambiguation_pipeline
+from src.pdf_parser.pdf_utils.ocr import (
     OCRProcessor,
-    PostProcessor,
 )
+from src.pdf_parser.pdf_utils.postprocess_layout import postprocessing_pipline
 
 CDN_DOMAIN = os.environ["CDN_DOMAIN"]
 
-_LOGGER = logging.getLogger(__file__)
+_LOGGER = logging.getLogger(__name__)
+_LOGGER.setLevel(logging.DEBUG)
```

Comment:
I'm not sure we want to hard-code the logging level to `DEBUG` - I'll add that for a follow-up fix too